### PR TITLE
Create quiz

### LINKED ISSUE #166: quiz fragment using SRAlgo and database
revise a chosen section of words

### PR DESCRIPTION
**Pull request by AdrianUlrich**
Unfinished: waiting for #131
closes #166 
No tests because tests impossible until #166 finished

### CODE REVIEW

File: app/src/main/java/com/github/sdpsharelook/revision/RevisionQuizFragment.kt [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            }
+        lifecycleScope.launch {
+            viewModel.uiEvent.apply { TODO() }
+        }
+    }
+
+    private fun handleHelpToggle(view: View) {
+        if (showHelp) {
+            showHelp = false
+            view.findViewById<Button>(R.id.answerQualityButton0).setText(R.string.quality0)
```
Comment (edoulaX) [inline app/src/main/java/com/github/sdpsharelook/revision/RevisionQuizFragment.kt:44]: You could maybe put all the ids on a list.

**Review by edoulaX — APPROVED** [review]
Nice work!

File: app/src/main/java/com/github/sdpsharelook/revision/RevisionQuizFragment.kt (hunk 1 above)
Comment (AdrianUlrich) [reply app/src/main/java/com/github/sdpsharelook/revision/RevisionQuizFragment.kt:44]: That is clever

Comment (AdrianUlrich) [reply app/src/main/java/com/github/sdpsharelook/revision/RevisionQuizFragment.kt]: Done

**Review by rombirli — APPROVED** [review]

**Pull request merged**